docker/webpack-watch

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+
+npm install
+cd frontend
+
+exec $(npm bin)/webpack --colors --watch --progress
```

Comment:
For a dev environment, I think it makes sense to launch the webpack process by default